client/src/components/core/Modals/Modals.module.css

```diff
@@ -0,0 +1,55 @@
+
+.modalBtnNewCol {
+    margin: 1rem 0;
+    background-color: aquamarine;
+    border: 5px solid green;
+
+}
+
+.modalDiv {
+    font-family: var(--font-regular);
+}
+
+.modalHeader {
+    font-size: var(--font-size-medium);
+    font-weight: bold;
+}
+
+.modalLabel {
+    font-size: var(--font-size-tiny);
+    color: var(--medium-grey);
+    font-weight: bold;
+}
+
+.modalInput {
+    font-size: var(--font-size-tiny) !important;
+    color: var(--black) !important;
```

Comment:
Rule of thumbs, _usually_ we don't want to use `!important` unless is the very last resource. We tried implementing whatever CSS rule we need it and nothing else worked ;)

Reply:
Ok, good to know! I couldn't figure out another way to update the font-size, I can post in the slack what it's looking like. I'd love advice for how to override the classes and get it working.

Reply:
nah, I think if you tried other things, this should be fine. The problem comes when this clashes with some other rule, which is why generally we don't want to use it. But given that this is just a small project, I think this should be fine :)

Reply:
So I went to take screen shots, and looks like it works without the !important. I went ahead and took it out. Thank you for the feedback!